metrics/prometheus/manifests/deployment.yaml

```diff
@@ -33,7 +33,7 @@ spec:
         image: prom/prometheus:v2.6.1
         imagePullPolicy: IfNotPresent
         args:
-          - '--storage.tsdb.retention=14d'
+          - '--storage.tsdb.retention=8h'
```

Comment:
Haven't tested it yet (time to sleep). Will do that tomorrow if you haven't already.

Reply:
This is probably impractically small, but let's try it to see if it takes effect.